deploy/helm/metrics-collector-config.yaml

```diff
@@ -770,6 +770,15 @@ processors:
           - action: aggregate_labels
             label_set: []
             aggregation_type: sum
+      - include: k8s.kubernetes_build_info
+        action: insert
+        match_type: regexp
+        experimental_match_labels: {"exported_job": ".*apiserver.*"}
+        new_name: sw.k8s.cluster.build
```

Comment:
I think your intent is to create `sw.k8s.cluster.build` attribute, but by this transformation you are creating the metric